docs/src/token.md

```diff
@@ -922,15 +922,16 @@ Vesting contract creator chooses various options upon creation, such as:
 - recipient
 - exact start and end date
 - (optional) cliff date and amount
-- (optional) release frequency 
+- (optional) release frequency
+
 Coming soon:
 - whether or not a contract is transferable by creator/recipient
 - whether or not a contract is cancelable by creator/recipient
 - subject/memo
 
-Timelock features can be integrated on different levels:
-- Program code: https://github.com/streamflow-finance/timelock
-- Rust SDK: https://crates.io/crates/streamflow-timelock ([source](https://github.com/streamflow-finance/timelock-crate))
+Resources:
+- Audit: Report can be found [here.](https://github.com/StreamFlow-Finance/timelock/blob/community/TIMELOCK_IMPLEMENTATION_COMMUNITY_REPORT_FINAL.pdf)
```

Comment:
You may want to substitute or add the crate timelock report, since that's actually looking at the program logic: https://github.com/StreamFlow-Finance/timelock-crate/blob/master/TIMELOCK_COMMUNITY_REPORT_FINAL.pdf

Reply:
You're right!
Added, thank you! 🙏